pep-0655.rst

```diff
@@ -14,16 +14,21 @@ Post-History: 31-Jan-2021, 11-Feb-2021, 20-Feb-2021, 26-Feb-2021, 17-Jan-2022, 2
 Abstract
 ========
 
-:pep:`589` defines syntax
-for declaring a TypedDict with all required keys and syntax for defining
+:pep:`589` defines notation
+for declaring a TypedDict with all required keys and notation for defining
 a TypedDict with :pep:`all potentially-missing keys <589#totality>` however it
```

Comment:
```suggestion
a TypedDict with :pep:`all potentially-missing keys <589#totality>`; however, it
```

Needs a semicolon here

Reply:
Did apply a similar change to split the long sentence, that doesn't create as dramatic of a pause.